src/createjs.Graphics.decodeSVGPath.ts

```diff
@@ -0,0 +1,17 @@
+import Graphics from "./Graphics";
+import applyMixins from "./applyMixins";
+
+/**
+ * decodeSVGPath intended for createjs.Graphics class
+ *
+ * @todo make tyepscript aware of this mixin being applied to createjs.Graphics class,
+ * and remove the corresponding // @ts-ignore comments where `decodeSVGPath` is used.
+ */
+class DecodeSVGPathMixin {
+  decodeSVGPath(data: string) {
+    Graphics.init(this, data);
+    return this;
+  }
+}
+
+applyMixins(createjs.Graphics, [DecodeSVGPathMixin]);
```

Comment:
@zoltan-nz Here I use the mixin approach described in the typescript docs: https://www.typescriptlang.org/docs/handbook/mixins.html

Reply:
@jedateach

You can add the following declaration to the end of this file and looks it solves the the problem below.

```typescript
declare global {
    namespace createjs {
        interface Graphics {
            decodeSVGPath(data: string): Graphics;
        }
    }
}
```

Reply:
Reference: https://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation